### DISCUSSION TITLE
Compatibility issues with React 18 and React Hook Form 5.6?

### DISCUSSION THREAD

**Discussion by jasonmng** [react-hook-form/react-hook-form#8623] [Q&A] [answered]
I'm seeing some validation issues and am wondering if there's a compatibility issue with React 18 and React Hook Form 5.6.  When I have a field with 

```
ref={register({ required: '...' })}
```

if I hit submit on the form, the form will not submit because of the required requirement but the error object returned from useForm() is empty. So it looks like the validation is happening but for some reason errors is not getting a proper value.  How might I debug this if it's not a compatibility problem? This code worked in React 17 and 16.

**Comment by bluebill1049** [comment, marked as answer]
Please move to react hook form V7, or disable strict mode in development for v5.
